src/Stripe.net/Services/ExternalAccounts/StripeExternalAccountCardUpdateOptions.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeExternalAccountCardUpdateOptions : INestedOptions
```

Comment:
I needed an extra class because:
* you can only update some properties
* those are sent as a top level parameter and not under `external_account`